roles/create_cluster/defaults/main.yml

```diff
@@ -20,8 +20,25 @@ HTTP_PROXY: ""
 HTTPS_PROXY: ""
 NO_PROXY: ""
 
-manifest_templates:
-  - 50-worker-nm-fix-ipv6.yml
-  - 50-worker-remove-ipi-leftovers.yml
-  - 02-fix-ingress-config.yml
-  - 01-master-node-scheduler.yml
+create_cluster__single_node_openshift_enabled: "{{ single_node_openshift_enabled | default(false) }}"
```

Comment:
Fair point! Let's discuss it with the team then.

One additional way would be to nest the vars under a key, like this:
```yaml
create_cluster:
  single_node_openshift_enabled: "{{ single_node_openshift_enabled | default(false) }}"
  high_availability_mode: "{{ create_cluster__single_node_openshift_enabled | ternary('None', 'Full') }}"
```
And then we could use `create_cluster.single_node_openshift_enabled` or even `create_cluster_vars.single_node_openshift_enabled`.

This might be a little bit more readable than `__`, as Ansible/jinja2 makes heavy use of the `.` notation.

Reply:
+1 for using a key and then dot notation. I think the `_vars` is slightly too verbose for my taste.

Reply:
I am alright with a single `_` for var names. `__` is tough to see and we should avoid it. Lastly Python and Jinja do strange things with dashes and fail to expand sometimes.

Reply:
I think it'd be best to move this conversation about style to a separate discussion. I've reverted the var prefixing change, all variables should now match the naming convention we've been using so far.